paths/job_template_locales/create.yaml

```diff
@@ -73,6 +73,20 @@ requestBody:
               type: string
             example:
             - abcd1234cdef1234abcd1234cdef1234
+          translator_team_ids:
+            description: Array of team ids to be assigned to the job locale as translators
+            type: array
+            items:
+              type: string
+            example:
+            - abcd1234cdef1234abcd1234cdef1234
+          reviewer_team_ids:
+            description: Array of team ids to be assigned to the job locale as reviewers
+            type: array
+            items:
+              type: string
+            example:
+            - abcd1234cdef1234abcd1234cdef1234
         required:
         - locale_id
-x-cli-version: '2.5'
+x-cli-version: '2.6.7'
```

Comment:
Should be 2.6.8 now as 2.6.7 has been released this morning :sweat_smile:

Reply:
Will update :)